<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detail</class>
 <widget class="QWidget" name="detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>191</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QLineEdit" name="etaGearInput"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="speedInput"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="KV_input"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="currInput"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="rtoInput"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="voltInput"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="resInput"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>电压/V</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>电流上限/A</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>电机kv值</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>电机内阻/Ω</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>减速比1：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>减速组效率%</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>空速/(m/s)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="analyzeButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>分析</string>
   </property>
  </widget>
  <widget class="QLabel" name="detailLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>161</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="propButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择螺旋桨</string>
   </property>
  </widget>
  <widget class="QLabel" name="adviceLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>410</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
